src/models/Offer.js

```diff
@@ -70,18 +62,22 @@ export type OfferOrder = {
   setTokenAllowance?: boolean,
   provider?: string,
   transactionPayload: TransactionPayload,
-}
+};
 
 export type ExchangeSearchRequest = {
   fromAssetCode: string,
   toAssetCode: string,
   fromAmount: number,
-}
+};
 
 export type Allowance = {
   provider: string,
   fromAssetCode: string,
   toAssetCode: string,
   transactionHash: string,
   enabled?: boolean,
-}
+};
+
+export type Allowances = {
```

Comment:
Consider renaming to `AccountAllowances` or `AllowancesPerAccount` for readability.